index.html

```diff
@@ -26,6 +26,224 @@ <h1>Hey There. I’m Alex</h1>
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button" onclick="#">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>  
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button" onclick="#">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button" onclick="#">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
```

Comment:
- [OPTIONAL] Kindly the button must include the all div for a particular project. 
Your design
![Screenshot from 2022-07-02 22-35-40](https://user-images.githubusercontent.com/77842237/177014316-4a404486-f6b4-4f16-81f6-6abb61eed473.png)

Figma design
![Screenshot from 2022-07-02 22-42-55](https://user-images.githubusercontent.com/77842237/177014324-001110ab-9975-4f0f-afa6-c2bb8258385d.png)